### PR TITLE
[Core] Adding tangent matrix definition to mortar utilities

### PR DESCRIPTION
**Pull request by loumalouomega**
Besides I defined epsilon as constexpr

### CODE REVIEW

**Comment by loumalouomega** [comment]
@roigcarlo For some reason Travis compiles and run correctly the tests, but returns false

**Comment by loumalouomega** [comment]
Ok, is not 100% OK, says OK... but

    OK (skipped=3, expected failures=1)

File: kratos/utilities/mortar_utilities.h [hunk 1]
```diff
@@ -384,7 +384,7 @@ class MortarUtilities
         
         const double this_norm = norm_2(normal);
         
-        KRATOS_DEBUG_ERROR_IF(this_norm < ZeroTolerance) << "Zero norm normal vector. Norm:" << this_norm << std::endl;
+        KRATOS_DEBUG_ERROR_IF(this_norm < std::numeric_limits<double>::epsilon()) << "Zero norm normal vector. Norm:" << this_norm << std::endl;
```
Comment (RiccardoRossi) [inline kratos/utilities/mortar_utilities.h]: accepting but...why don'y you use "zero_tolerance" here too?

File: kratos/utilities/mortar_utilities.h [hunk 2]
```diff
@@ -384,7 +384,7 @@ class MortarUtilities
...
         
         normal /= this_norm;
```
Comment (RiccardoRossi) [inline kratos/utilities/mortar_utilities.h]: what happens if the norm is zero

**Review by RiccardoRossi — DISMISSED** [review]

File: kratos/utilities/mortar_utilities.h (hunk 2 above)
Comment (loumalouomega) [reply kratos/utilities/mortar_utilities.h]: It will throw an error in debug

File: kratos/utilities/mortar_utilities.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/utilities/mortar_utilities.h]: No, because is used just once

**Review by roigcarlo — APPROVED** [review]

**Pull request merged**